gtsam/linear/AcceleratedPowerMethod.h

```diff
@@ -0,0 +1,171 @@
+/* ----------------------------------------------------------------------------
+
+ * GTSAM Copyright 2010-2019, Georgia Tech Research Corporation,
+ * Atlanta, Georgia 30332-0415
+ * All Rights Reserved
+ * Authors: Frank Dellaert, et al. (see THANKS for the full author list)
+
+ * See LICENSE for the license information
+
+ * -------------------------------------------------------------------------- */
+
+/**
+ * @file   AcceleratedPowerMethod.h
+ * @date   Sept 2020
+ * @author Jing Wu
+ * @brief  accelerated power method for fast eigenvalue and eigenvector
+ * computation
+ */
+
+#pragma once
+
+// #include <gtsam/base/Matrix.h>
+// #include <gtsam/base/Vector.h>
+#include <gtsam/linear/PowerMethod.h>
+
+namespace gtsam {
+
+using Sparse = Eigen::SparseMatrix<double>;
+
+/* ************************************************************************* */
+/// MINIMUM EIGENVALUE COMPUTATIONS
+
+// Template argument Operator just needs multiplication operator
+template <class Operator>
+class AcceleratedPowerMethod : public PowerMethod<Operator> {
+  /**
```

Comment:
doxygen comments should go right above class. Incorporate template argument doc there.